e2e/rbd_helper.go

```diff
@@ -6,7 +6,10 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+	"sync"
+	"time"
 
+	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
```

Comment:
Please use `volumesnapshot` instead of `v1beta1` in the code. When we move to a newer version, we would not need to update the `v1beta1` everywhere.